fastchat/serve/monitor/clean_battle_data.py

```diff
@@ -35,33 +36,23 @@
     "lamda",
     "google",
     "llama",
+    "qianwan",
+    "alibaba",
+    "mistral",
+    "zhipu",
+    "KEG lab",
+    "01.AI",
+    "AI2",
+    "Tülu",
```

Comment:
tulu?

Reply:
the model seems to be trained to output `Tülu` not `Tulu`.
<img width="549" alt="image" src="https://github.com/lm-sys/FastChat/assets/1206058/b1ac5275-f4a9-455a-b74b-cadebc3201da">

Reply:
Or we can add both